ios-guide.md

```diff
@@ -42,7 +42,7 @@ Add the end of this step, your Xcode config should look like this:
 ### Modify your app to respond to the URL scheme
 
 - Open `AppDelegate.m`
-- Add an import: `#import "RNGoogleSignin.h"`
+- Add an import: `#import <RNGoogleSignin/RNGoogleSignin.h>`
```

Comment:
I know this sounds weird, but the `Add an import: #import "RNGoogleSignin.h"` information was added 2 months ago, so this probably works for some people. Can you make sure both imports are mentioned, saying something that makes people try one, and then the other if the first one does not work? Thanks.